lib/py/reddit_edgecontext/__init__.py

```diff
@@ -33,6 +34,7 @@
 
 
 COUNTRY_CODE_RE = re.compile(r"^[A-Z]{2}$")
+LOCALE_CODE_RE = re.compile(r"^[a-z]{2}_[A-Z]{2}$")
```

Comment:
🔕 this means we only allow `en_US` and not just `en`. is that intended? I think there are cases that you can only get the language without country.

Reply:
You're correct. Updated to catch both cases of lang_REGION and just lang. There are also valid subregion codes that currently won't pass this check, so we may need to expand on this when we add support for them (e.g. https://en.wikipedia.org/wiki/UN_M49). I didn't want to overcomplicate things for now.